notebooks/misc/dp_mixgauss_truncated.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">For all figures, please do the following.
A tht eopt of the notebook add this

try:
    from probml_utils import savefig, latexify
except ModuleNotFoundError:
    %pip install -qq git+https://github.com/probml/probml-utils.git
    from probml_utils import savefig, latexify


</pre><p> </p><p>Then do the following</p><pre class="ql-syntax" spellcheck="false">savefig("dp_stick_breaking_samples") 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/924/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>